### DISCUSSION TITLE
what diff about responsive and fluid option?

### DISCUSSION THREAD

**Discussion by startjava** [videojs/video.js#8675] [Q&A]
### Description

what diff about responsive and fluid option?

### Reduced test case

?

### Steps to reproduce

1.
2.
3.

### Errors

?

### What version of Video.js are you using?

?

### Video.js plugins used.

?

### What browser(s) including version(s) does this occur with?

?

### What OS(es) and version(s) does this occur with?

?

**Comment by mister-ben** [comment]
https://videojs.com/guides/layout/

Fluid mode resizes the player maintaining an aspect ratio. Responsive changes the controls layout/size to fit the player size.
